File: Basic/Loop/219432Q.c
#include <stdio.h>
#define intll long long

void solution() {

	int n;
	scanf("%d", &n);

	if (n == 0) {
		printf("0\n");
		return;
	}

	while (n) {
		printf("%d ", n % 10);
		n /= 10;
	}
	printf("\n");
}

int main() {
	char TEST_CASE = 1;
	int t = 1;
	if (TEST_CASE == 1) {
		scanf("%d", &t);
	}

	while (t--) {
		solution();
	}
	return 0;
}

File: Basic/Loop/219432Q.go
package main

import (
	"fmt"
)

func solution() {

	var n int
	fmt.Scan(&n)

	if n == 0 {
		fmt.Println(n)
		return
	}

	for n > 0 {
		fmt.Printf("%d ", n % 10)
		n /= 10
	}

	fmt.Println()
}

func main() {
	TEST_CASE := true
	t := 1
	if TEST_CASE {
		fmt.Scanf("%d", &t)
	}

	for t != 0 {
		solution()
		t--
	}
}
